modules/rgbd/include/opencv2/rgbd/volume.hpp

```diff
@@ -50,9 +49,66 @@ enum class VolumeType
     HASHTSDF = 1
 };
 
-CV_EXPORTS_W cv::Ptr<Volume> makeVolume(VolumeType _volumeType, float _voxelSize, cv::Matx44f _pose,
-                           float _raycastStepFactor, float _truncDist, int _maxWeight,
-                           float _truncateThreshold, Vec3i _resolution);
+struct CV_EXPORTS_W VolumeParams
+{
+    /** @brief Type of Volume
+        Values can be TSDF (single volume) or HASHTSDF (hashtable of volume units)
+    */
+    VolumeType type;
+
+    /** @brief Resolution of voxel space
+        Number of voxels in each dimension.
```

Comment:
"Applicable only for single volume TSDF"

Reply:
Done.